rest/channel_test.go

```diff
@@ -0,0 +1,44 @@
+package rest_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Channel", func() {
+	var (
+		event   string
+		message string
+	)
+
+	BeforeEach(func() {
+		event = "sendMessage"
+		message = "A message in a bottle"
+	})
```

Comment:
It looks like these variables don't change, so I would drop these assignments and make them constants:

```go
const (
	event   = "sendMessage"
	message = "A message in a bottle"
)